website/content/en/docs/Getting Started/v0.2.0/notifications.md

```diff
@@ -0,0 +1,116 @@
+---
```

Comment:
only this file is needed, and the path should be `website/content/en/docs/Getting Started/v0.3.0/notifications.md`